docs/manual/troubleshooting.tex

```diff
@@ -764,14 +764,28 @@ \section{Building from source\label{build-source}}
 \subsection{Install prerequisites\label{building-prerequisites}}
 
 You need to install several packages in order to build the Checker
-Framework.  The following command installs them for Ubuntu:
+Framework. The following command installs them for Ubuntu:
 
 \begin{Verbatim}
 sudo apt-get update
 sudo apt-get install --yes ant dia git hevea junit4 librsvg2-bin make mercurial openjdk-8-jdk \
  texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended unzip
 \end{Verbatim}
 
+% Building Checker Framework from source on Mac OS
+
+If you are building on Mac OS, then the following commands install the aforementioned packages:
+
+\begin{Verbatim}
+brew update
+brew install caskroom/cask/brew-cask
+brew install git ant hevea mercurial librsvg unzip make
+brew cask install java
+brew cask install mactex
+\end{Verbatim}
+
+Mac OS users also need to copy \<hevea.sty> into \<\$HOME/Library/texmf/tex/latex> (you probably need to make that directory first).
```

Comment:
Please make this a concrete pair of commands:
```
mkdir -p $HOME/Library/texmf/tex/latex
cp -p .../hevea.sty $HOME/Library/texmf/tex/latex
```

It's generally best to give concrete commands that can be cut-and-pasted, rather than a general description, which requires more effort from users and is prone to misinterpretation.